### PR TITLE
Chore/move gui to modules

### PR DESCRIPTION
**Pull request by chmelevskij**
* refactor GUI to be class based
* use import instead of global where possible
* use esm module for `gui.js`

### CODE REVIEW

File: src/main.html
Comment (chmelevskij) [inline src/main.html:95]: Anything what needed initialisation gets called with import of `main.js`

File: src/tabs/presets/presets.js
Comment (chmelevskij) [inline src/tabs/presets/presets.js:1]: Modules are in strict mode by default

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
I think I don't receive a mail when changed from draft to ready to review. It seems ok to me.

**Comment by haslinghuis** [comment]
Android build is failing as other builds are cancelled.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
